tests/gtests/internals/test_sdpa.cpp

```diff
@@ -963,7 +979,7 @@ void prim_sdpa_quant(const sdpa_dims_t &p, const sdpa_tensors_t &t,
     auto bmm1_prim = matmul(bmm1_pd);
 
     primitive_attr softmax_attr;
-    softmax_attr.set_scratchpad_mode(scratchpad_mode::user);
+    softmax_attr.set_scratchpad_mode(scratchpad_mode::library);
```

Comment:
Is it necessary? Why choose `library` over `user`?

Reply:
For my cases the scratch space was always zero and this was complicating the code so I removed it. I don't think it affects the testing.